docroot/modules/custom/cgov_glossary/tests/src/Unit/GlossaryAppModulePluginTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Drupal\Tests\cgov_glossary\Unit;
+
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Tests for the PathProcessorAppModule.
+ */
+class GlossaryAppModulePluginTest extends UnitTestCase {
+
+  /**
+   * Tests GlossaryPlugin functions.
+   */
+  public function testGlossaryPlugin() {
+    $this->assertTrue(TRUE);
```

Comment:
You could at least test the matchRoute function here...

See https://github.com/NCIOCPL/cgov-digital-platform/blob/630b2f5fbe3afbb0780c6bc250c4e64574312bd4/docroot/modules/custom/app_module/tests/src/Functional/AppModulePluginBaseTest.php

Reply:
Tests for matchRoute are included in this file now.